src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -303,4 +303,7 @@
     <value>PackageSourceMapping is enabled and there are multiple package sources associated with the same key(s): {0}. Path: {1}</value>
     <comment>0 - package sources separated with comma 1 - path</comment>
   </data>
+  <data name="ShowError_ConfigDuplicateDisabledSources" xml:space="preserve">
+    <value>Cannot update package sources. NuGet.Config contains a duplicated key in the disabled package sources section.</value>
```

Comment:
Can we include the file path of the config?

IMO, we should refer to it as `NuGet.Config` either, but rather a configuration file. 

`NuGet.Config` implies we know the exact casing for the while which isn't necessarily true. 

While the overall changes is for VS, the API change is for all platforms.

Reply:
I changed the message, I added the path and duplicated key
![image](https://user-images.githubusercontent.com/43253759/199296717-37baa1b6-0f11-4288-a3d3-80f306b6ae67.png)